### PR TITLE
DOXYGEN related parts are commented out

### PR DESCRIPTION
**Pull request by KiSooLee**
DOXYGEN related parts are commented out

### CODE REVIEW

**Comment by KiSooLee** [comment]
+1

File: CMakeLists.txt [hunk 1]
```diff
@@ -162,22 +162,21 @@ else()
   add_definitions(-D'SVN_VERSION=\"\"')
 endif()
 
-
 # setup doxygen
-find_package(Doxygen)
```
Comment (davidlt) [inline CMakeLists.txt:167]: It's better to add CMake option to disable documentation.

File: cmake_modules/FindCLHEP.cmake [hunk 2]
```diff
@@ -1,5 +1,6 @@
 Find_Program(CLHEP_CONFIG_EXE NAMES clhep-config PATHS
-	$ENV{CLHEP_PARAM_PATH}/bin
+#	$ENV{CLHEP_PARAM_PATH}/bin
+	$ENV{CLHEPDIR}/bin
```
Comment (davidlt) [inline cmake_modules/FindCLHEP.cmake]: If you are changing CLHEP patch, change it everywhere.

File: env.log [hunk 3]
```diff
@@ -0,0 +1,156 @@
+declare -x CASCADE_PDFPATH="/cvmfs/cms.cern.ch/slc6_amd64_gcc491/external/cascade/2.2.04-kpegke2/share"
```
Comment (davidlt) [inline env.log]: Have no idea why this file would be needed, remove.

File: cmake_modules/FindCLHEP.cmake (hunk 2 above)
Comment (KiSooLee) [inline cmake_modules/FindCLHEP.cmake]: my mistake.
I will keep using $ENV{CLHEP_PARAM_PATH}
Let me modify

File: env.log (hunk 3 above)
Comment (KiSooLee) [inline env.log]: This file was used to find the path of CLHEP let me remove it

**Comment by KiSooLee** [comment]
Hi, is this pull request reasonable?
Should I squash these two commit or need more modification?

**Pull request merged**